pkg/util/mount/mount.go

```diff
@@ -82,9 +82,9 @@ type HostUtils interface {
 	// MakeDir creates a new directory.
 	// Will operate in the host mount namespace if kubelet is running in a container
 	MakeDir(pathname string) error
-	// Will operate in the host mount namespace if kubelet is running in a container.
```

Comment:
do we still want this information in the comment?

Reply:
We're preparing this package to be exported out to k8s/utils, and the review comments recommended removing K8s-specific information from the interface.

Although in this PR, you may just want to add a the new description to hopefully avoid merge conflicts with https://github.com/kubernetes/kubernetes/pull/81180/